### PR TITLE
Fixing and merging functions to let saveCard function work and validate.

### PR DESCRIPTION
**Pull request by hborrelli1**
Merge mine and Jeremiahs functions to validate the input and populate the card. 

We had a couple naming conflicts, so I refactored a little and changed some names to keep things more consistent.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,16 +68,12 @@ <h3>Backgrounds</h3>
         </div>
         <form class="save-outfit" action="#" method="#">
           <input id="saveOutfitInput" class="outfit-namer"type="text" name="outfitName" value="" placeholder="name this outfit">
-          <button id="saveOutfitBtn" class="button-style disable-save" type="button" name="saveOutfit">Save</button>
+          <button id="saveOutfitBtn" class="button-style disable-save" type="button" name="saveOutfit" disabled>Save</button>
```
Comment (hborrelli1) [inline index.html:71]: I set the input to disabled by default.

File: index.html [hunk 2]
```diff
@@ -68,16 +68,12 @@ <h3>Backgrounds</h3>
...
         </form>
       </section>
       <section class="column3 body-column">
         <h3>Saved outfits</h3>
         <section class="outfits-list">
-          <button class="button-style">Fancy<img src="assets/close.svg" alt="Close"></button>
-          <button class="button-style">A Day Out<img src="assets/close.svg" alt="Close"></button>
-          <button class="button-style">Beach Time<img src="assets/close.svg" alt="Close"></button>
-          <button class="button-style">Mars Prom<img src="assets/close.svg" alt="Close"></button>
```
Comment (hborrelli1) [inline index.html:80]: I removed the placeholder cards now that the save card functionality is working.

File: main.js [hunk 3]
```diff
@@ -8,18 +9,21 @@ var backgroundsSection = document.querySelector('.backgrounds-list');
 var hatBtns = document.querySelectorAll('.hats-list .button-style');
 var garmentAppear = document.querySelectorAll('.garment');
 //next two variables allow us to use clearInputField function
-var saveOutfit = document.querySelector('.disable-save');
+var saveOutfitBtn = document.getElementById('saveOutfitBtn');
 var outFitInput = document.querySelector('.outfit-namer');
```
Comment (hborrelli1) [inline main.js:13]: I renamed these variables and adjusted in the event listeners

File: main.js [hunk 4]
```diff
@@ -106,25 +110,31 @@ function changeBackgroundImg() {
...
   if(outFitInput.value != "") {
-    console.log('yo');
-    saveOutfit.classList.remove('disable-save');
-  } else(saveOutfit.classList.add('disable-save'))
+    saveOutfitBtn.classList.remove('disable-save');
+    saveOutfitBtn.removeAttribute('disabled');
+  } else {
+    saveOutfitBtn.classList.add('disable-save')
+    saveOutfitBtn.setAttribute('disabled', '');
+  }
```
Comment (hborrelli1) [inline main.js:125]: I added code to change the disabled attribute of the input so it doesn't create card without any text

**Comment by CLPena** [comment]
Harry, this looks great! Thanks for commenting on specific lines to explain what changes you made and why. Super easy to understand. I'll merge it now.

**Pull request merged**